client/keys/add.go

```diff
@@ -49,7 +52,9 @@ Use the --pubkey flag to add arbitrary public keys to the keystore for construct
 		Args: cobra.ExactArgs(1),
 		RunE: runAddCmd,
 	}
-	cmd.Flags().String(FlagPublicKey, "", "Store only a public key (useful for constructing multisigs e.g. cosmospub1...)")
+	cmd.Flags().StringSlice(flagMultisig, nil, "Construct and store a multisig public key key. It implies --pubkey")
+	cmd.Flags().Uint(flagMultiSigThreshold, 1, "K out of N required signatures. For use in conjunction with --multisig")
+	cmd.Flags().String(FlagPublicKey, "", "Store only a public key")
```

Comment:
I think this flag's description could be improved. What exactly is the user providing? Name of the pubkey? The pubkey itself?

Reply:
Amended, please review.